### ISSUE TITLE
Elevated plasma hexanoylcarnitine, C6:0

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5963]
**Preferred term label:** Elevated plasma hexanoylcarnitine, C6:0

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Hexanoylcarnitine concentration in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
